ilc/client.js

```diff
@@ -95,9 +95,13 @@ document.addEventListener('click', function (e) {
         return;
```

Comment:
Here also check for `e.defaultPrevented === false`

Reply:
Updated